Changing npm scope breaks "Reset focus" script

When changing npm scope by running `npm run config`, it seems that a global search + replace is made but there is an incorrectly replaced line: https://github.com/NativeScript/plugin-seed/blob/ecac7537937edf86e9d178e5ade2beb263fb382f/tools/workspace-scripts.js#L75

The line is replaced like this:

```diff
-        script: 'nx g @nativescript/plugin-tools:focus-packages',
+        script: 'nx g @myscope/plugin-tools:focus-packages',
```

And that breaks the reset command 🤷 

```
npm start
> plugins@0.0.0 start
> nps

nps is executing `default` : nps-i
? NativeScript Plugins ~ made with ❤️  Choose a command to start... focus.reset                    Reset Focus
nps is executing `focus.reset` : nx g @myscope/plugin-tools:focus-packages

 NX   Unable to resolve @myscope/plugin-tools:focus-packages.

Cannot find module '@myscope/plugin-tools/package.json'
Require stack:
- /Users/myuser/projects/test-plugins/node_modules/nx/src/utils/package-json.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/project-graph/plugins/loader.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/project-graph/plugins/internal-api.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/project-graph/project-graph.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/project-graph/file-utils.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/utils/package-manager.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/utils/child-process.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/command-line/migrate/command-object.js
- /Users/myuser/projects/test-plugins/node_modules/nx/src/command-line/nx-commands.js
- /Users/myuser/projects/test-plugins/node_modules/nx/bin/init-local.js
- /Users/myuser/projects/test-plugins/node_modules/nx/bin/nx.js
Pass --verbose to see the stacktrace.

The script called "focus.reset" which runs "nx g @myscope/plugin-tools:focus-packages" failed with exit code 1 https://github.com/Markforged/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
```

--- a/packages/plugin-tools/src/generators/config/index.ts
+++ b/packages/plugin-tools/src/generators/config/index.ts
@@ -102,7 +102,8 @@ export default async function (tree: Tree, schema: Schema) {
 
   const workspaceScriptsPath = `tools/workspace-scripts.js`;
   let workspaceScripts = tree.read(workspaceScriptsPath).toString('utf-8');
-  workspaceScripts = workspaceScripts.replace(/@nativescript/gm, `${customNpmScope}`);
+  workspaceScripts = workspaceScripts.replace(/@nativescript/gm, `${customNpmScope}`)
+    .replace(new RegExp(`${customNpmScope}/plugin-tools:`, 'gm'), `@nativescript/plugin-tools:`);
   // context.logger.info(travisContent);
   tree.write(workspaceScriptsPath, workspaceScripts);
 

